samcli/local/apigw/local_apigw_service.py

```diff
@@ -251,13 +251,10 @@ def _construct_event(flask_request, port, binary_types):
         identity = ContextIdentity(source_ip=flask_request.remote_addr)
 
         endpoint = PathConverter.convert_path_to_api_gateway(flask_request.endpoint)
-        method = flask_request.method
```

Comment:
Removed those variables because of:

> R:243, 4: Too many local variables (17/15) (too-many-locals)

Updating maximum number of locals for function / method body to:

> max-locals=17